app/templates/newsletter.hbs

```diff
@@ -1,4 +1,5 @@
 <Newsletter::NewsletterHeaderOverview
-  @meeting={{this.model.meeting}}
+  @meeting={{@model.meeting}}
+  @agenda={{@model.agenda}}
```

Comment:
Do we need both arguments?  Meeting can be uniquely derived from the agenda?
Argument documentation of `Newsletter::NewsletterHeaderOverview` needs to be updated